### DISCUSSION TITLE
Share your experience

### DISCUSSION THREAD

**Discussion by orlangure** [orlangure/gnomock#88] [Show and tell]
Hi there!😼

Gnomock is a young project, it doesn't have many users, and due to its nature (a package for testing...) it's hard to see how it is being used in the wild.

If you use Gnomock at work or in a personal project, it would be very interesting to learn from your experience.

Below are a few questions that can help figuring out what exactly to share. Feel free to answer some of them, if you wish😺
- Do you use presets? Which ones?
- Do you use initial state setup features or available presets (SQL schema, seed data, etc.)?
- Have you tried implementing custom presets for your particular needs? How did it go?
- How stable are the tests you wrote using Gnomock?

Feel free to include links to projects that use Gnomock for tests😻

**Comment by pawndev** [comment]
Hello ! I'm using gnomock at work, so I have no repositories to share. But here are my feelings about this tool:
- Yep, we're using postgres and localstack preset (actually, working to integrate Redis preset too).
- We use a database migration tool from code when gnomock up our postgres containers.
- I already implemented another postgres preset, when I was waiting for the [timezone support](https://github.com/orlangure/gnomock/pull/293). And it was smooth to do this.
- Our test are really stable, nothing to complain !

So far, it's a pleasant tool to work with, thanks again for this library 🙏
